spec/generators/rspec/install/install_generator_spec.rb

```diff
@@ -27,7 +27,7 @@ def require_rspec_rails
     match(/^require 'rspec\/rails'$/m)
   end
 
-  def use_transactional_fixtures
+  def have_transactional_fixtures
```

Comment:
This is just to prevent a redefined method warning. I guess this conflicts with the original `use_transactional_fixtures`.